### ISSUE TITLE
Typeface update

### ISSUE THREAD

**Issue by omnat** [MetaMask/Design#175]
Typeface update (new) - Something @rachelcope and @cjeria have been looking at is updating the app's typeface. We're really keen on Circular and it might be a good place to start introducing it in the DS.

Pair with @rickycodes to test out this font in an extension branch

**Comment by cjeria** [comment]
@rickycodes here's the font-family we could use to do a test of how it would look!

[circularStd - fontfamily.zip](https://github.com/MetaMask/Design/files/4073144/circularStd.-.fontfamily.zip)

**Comment by cjeria** [comment]
@rickycodes @estebanmino I also received the **regular** weight version of the font that is not included in the zip above. Their formats are .otf, and the regular font (linked below) is .woff. Which one is best for apps? I could request all to be .woff if that's better? 

[CircularXXWebTest-Regular.woff.zip](https://github.com/MetaMask/Design/files/4118981/CircularXXWebTest-Regular.woff.zip)

**Comment by estebanmino** [comment]
@cjeria  Is this a final call or you want to try it out on the app?

**Comment by cjeria** [comment]
@estebanmino yes, we'd like to test this new font to make sure nothing looks too weird in the UI before we invest $$$ to buy it.

**Comment by estebanmino** [comment]
@cjeria  should I put this on my to do next thing to work on? since you have IOS and @rickycodes doesn't.

**Comment by cjeria** [comment]
That would be great @estebanmino! 

I believe @rickycodes is able to test the new font in the extension?

**Comment by rickycodes** [comment]
yep! I can test it in the extension

**Comment by cjeria** [comment]
Just reviewed new fonts in the extension and it's looking really good. I'm confirming with the font designer which font formats we'd be receiving for the app. The formats Ricky suggested are found in this [article](https://css-tricks.com/snippets/css/using-font-face/#article-header-id-0) in the first syntax block on that page.

During testing, I also discovered pages and sections in the extension that didn't have the new typeface updated. Ricky wants to refactor the web app so that styles cascade properly. Let's make sure we allocate time to refactoring the CSS cc @omnat 

I'll set up some time with @rachelcope  to preview the new typeface in the extension next.
